plugin/PDFView.qml

```diff
@@ -190,7 +198,41 @@ SilicaFlickable {
 
     children: [
         HorizontalScrollDecorator { color: Theme.highlightDimmerColor },
-        VerticalScrollDecorator { color: Theme.highlightDimmerColor }
+        VerticalScrollDecorator { color: Theme.highlightDimmerColor },
+        PDFContextMenu {
```

Comment:
You're right, I've notice the PDFView (not being fullscreen) issue. But I've no clue how to deal with this. PDFDocumentPage is the element being fullscreen, but PDFContextMenu should be logically linked with PDFView (at least because callback may be methods of PDFView like ```jumpTo()```).

Do you think it's possible to access the ApplicationWindow (or similar), so the Rectangle of the PDFContextMenu can be set at x:0 and y:0 and width: window.width… ?